fastlane/Fastfile

```diff
@@ -54,6 +54,16 @@ platform :ios do
     )
   end
 
+  desc "Runs tests on select platforms for verifying pull requests"
+  lane :verify_pull_request do
+    verify(
+     sim: 'iPhone 6s (9.3)'
+    )
+    verify(
+     sim: 'iPad Air 2 (9.3)'
+    )
```

Comment:
I guess thats debatable? What's more valuable to test, the latest OS or the oldest supported?

Or if we can't test 9.3 on the Jenkins box then I guess 10.3 it is